package.json

```diff
@@ -23,7 +23,7 @@
     "lint:all": "yarn lint --ext .js .",
     "lint:fix-all": "yarn lint:all --fix",
     "build:all": "npm-run-all --print-name --parallel sass html",
-    "html": "cat ./ui/roll20_character_sheet_0.2.html > ./bin/main.html",
+    "html": "./ui/main.sh > ./bin/main.html",
```

Comment:
Using a shell script to compile the html into a single `main.html` file above.
This makes the code easier to manage, and more importantly, gives us the ability to lint and auto-format the inline Javascript in HTML.

I'll push a follow up PR for the Javascript lint and auto-formatting.
This allows us to catch errors in the editor, early. (We can even think about tests!)

Reply:
This is an example of auto-fixed JavaScript:
https://github.com/epic-power-rpg/roll20/pull/27/files